bika/lims/content/analysis.py

```diff
@@ -1760,7 +1759,9 @@ def workflow_script_submit(self):
             try:
                 workflow.doActionFor(self, "attach")
             except WorkflowException:
-                pass
+                logger.error(
+                    "attach action failed for analysis %s" % self.getId())
```

Comment:
Better like this:
```python
logger.error(
    'Workflow transition error: \'attach\' action failed for analysis {0}'.format(self.getId()))
```
see https://docs.plone.org/develop/styleguide/python.html#string-formatting